sphinx/themes/basic/static/basic.css_t

```diff
@@ -114,16 +114,25 @@ img {
 
 /* -- search page ----------------------------------------------------------- */
 
-ul.search {
-    margin: 10px 0 0 20px;
-    padding: 0;
-}
-
 ul.search li {
     padding: 5px 0 5px 20px;
-    background-image: url(file.png);
-    background-repeat: no-repeat;
-    background-position: 0 7px;
+    list-style: "\25A1";  /* Unicode: White Square */
```

Comment:
I wonder whether it might make sense to be slightly more explicit and define these using the CSS `list-style-type` property?

Also, to check: is the white-square default here intended to highlight cases where sites have upgraded their Sphinx CSS but not rebuilt their `searchindex.js`?

Reply:
I'm not a CSS expert. I was anticipating that `list-style` is easier to override for downstream themes, e.g. `list-style: url(image.svg)`, but just a guess. If you say `list-style-type` is better, I'm happy to change.

I've added the empty square as a catch-all fallback; e.g. could be that somebody rewrites search and includes a new type, but forgets the corresponding CSS. Likely other things can happen and the empty square as as placeholder is better than nothing.

Reply:
> I'm not a CSS expert. I was anticipating that `list-style` is easier to override for downstream themes, e.g. `list-style: url(image.svg)`, but just a guess. If you say `list-style-type` is better, I'm happy to change.

Me neither, and your intuition may be correct.  I based my suggestion on a sense that `list-style-type` is slightly less ambiguous (`list-style` _does_ accept the same values, but can also be set to an image URL), and also a small note on the [Mozilla MDN page](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style):

> > The `list-style` property is specified as one, two, or three values in any order. If [`list-style-type`](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-type) and [`list-style-image`](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-image) are both set, the `list-style-type` is used as a fallback if the image is unavailable.
 
> I've added the empty square as a catch-all fallback; e.g. could be that somebody rewrites search and includes a new type, but forgets the corresponding CSS. Likely other things can happen and the empty square as as placeholder is better than nothing.

That makes sense.  A concern I have with it is that the same (or a similar) symbol is used on some systems when a unicode codepage/symbol is not found in the relevant font, and that could make problems slightly trickier to track down.  Perhaps we could set the value to `initial` (initial browser default)?

Reply:
>  A concern I have with it is that the same (or a similar) symbol is used on some systems when a unicode codepage/symbol is not found in the relevant font

Our assumptions in the HTML search is that the user has a modern browser, and hence we should be confident in using Unicode -- every modern browser is able to display emoji, etc.